src/sentry/static/sentry/app/views/alerts/details/index.tsx

```diff
@@ -27,8 +31,17 @@ class IncidentDetails extends React.Component<Props, State> {
   state: State = {isLoading: false, hasError: false};
 
   componentDidMount() {
-    const {api, params} = this.props;
+    const {api, organization, params} = this.props;
+
+    trackAnalyticsEvent({
+      eventKey: 'alert_details.viewed',
+      eventName: 'Alert Details: Viewed',
+      org_id: parseInt(organization.id, 10),
```

Comment:
organization_id (in other places too). this gets us everytime. shouldn't typescript catch this?

Reply:
ah I think I initially used `analytics`

Reply:
Typescript doesn't catch this because it's defined in `getsentry` - it's also a bit confusing because it passes it to reload as `org_id`